Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -124,8 +124,14 @@ void IBocColumnRenderer.RenderTitleCell (BocColumnRenderingContext renderingCont
         if (!string.IsNullOrEmpty (columnItemID))
           renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.ItemID, columnItemID);
 
-        var columnTitle = StringUtility.NullToEmpty (renderingContext.ColumnDefinition.ColumnTitleDisplayValue);
-        renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags (columnTitle));
+        var columnTitle = renderingContext.ColumnDefinition.ColumnTitleDisplayValue;
+        var content = columnTitle.Type switch
+        {
+            WebStringType.Encoded => HtmlUtility.StripHtmlTags (columnTitle.GetValue()),
+            WebStringType.PlainText => columnTitle.GetValue(),
+            var t => throw new NotSupportedException ($"The WebStringType '{t}' is not supported."),
+        };
+        renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, content);
```

Comment:
We have a pattern for this using a ternary operator. Please update to use the established style for consistency reasons.

@a-ctor could you please add a reference to an existing sample?

Reply:
https://github.com/re-motion/Framework/pull/255/files#diff-6e902a3b4862807594bba6d3fe403de8c343545854dd3be2de8a3f83a4097091R209-R211

Reply:
Its just a `HtmlUtility.StripHtmlTags` overload

Reply:
no, I meant this code snippet:
```
        writer.AddAttribute (
            DiagnosticMetadataAttributes.Content,
            Text.Type == WebStringType.Encoded
                ? HtmlUtility.StripHtmlTags (Text.GetValue())
                : Text.GetValue());
```

I'm not even saying that switch expression or ternary is better, it's just a matter of consistency

Reply:
Extending the enum when using the pattern could lead to unwanted behavior, that's why I decided to use the switch expression. Anyway the snipped you posted is implemented as `HtmlUtility.StripHtmlTags` overload, that's what I'm using now.

Reply:
And I just realized, we already have the overload for HtmlUtility.StripHtmlTags integrated in the feature branch (@a-ctor thanks!)

So, @iam-mholle we can use
`renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags (columnTitle.GetValue()));`
and be done with it.

Reply:
And I just realized, we already have the overload for HtmlUtility.StripHtmlTags integrated in the feature branch (@a-ctor thanks!)

So, @iam-mholle we can use
`renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags (columnTitle));`
and be done with it. (NO `GetValue()` !!)